.github/workflows/ci.yml

```diff
@@ -153,6 +153,7 @@ jobs:
       run: |
         source activate pandas-dev
         pytest pandas/tests/frame/methods --array-manager
+        pytest pandas/tests/reshape/merge --array-manager
```

Comment:
Only `reshape/merge` and not `reshape/concat`, as there are too many tests still failing in the concat tests (because axis=0 is not yet handled properly)